### ISSUE TITLE
Kernel Gateway

### ISSUE THREAD

**Issue by pepgal** [jupyter/help#187]
Hi, 

i'm new on kernel gateway. I use that into Jupyter container started by Docker.
 There is a way to passing multiple  notebooks into  --KernelGatewayApp.seed_uri  command param? I have a notebook that call other notebook (as module) and I would to expose as rest client. Thanks

**Comment by parente** [comment]
There isn't a way to pass multiple notebooks at present. https://github.com/jupyter/kernel_gateway/issues/225 is tracking that feature request. Pull requests are welcome!

**Comment by pepgal** [comment]
Thanks Peter. I will watch it later. There is a way to push a module py (instead notebook) by DockerFile into py container service that exposing notebook/rest ? 

I tried by dockerFile :

`FROM jupyter/all-spark-notebook
USER root
RUN pip install jupyter-kernel-gateway
COPY data/notebooks/hello /tmp
RUN ls /tmp
RUN python  /tmp install
MAINTAINER Peppe Gallo <giuseppe.gallo@alliance-healthcare.net>
EXPOSE 4040`

I also tried by conda install command but my module isn't referenced by Kernel Gateway lib. Thanks!
